Description: fix a spelling mistake
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=85989
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=85989
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-10-11

--- a/lib/XML/Filter/Distributor.pm
+++ b/lib/XML/Filter/Distributor.pm
@@ -259,12 +259,12 @@
 
     $p->set_handlers( $handler1, $handler2 );
 
-Provided for compatability with other SAX processors, use set_handlers
+Provided for compatibility with other SAX processors, use set_handlers
 instead.
 
 =item set_handler
 
-Provided for compatability with other SAX processors, use set_handlers
+Provided for compatibility with other SAX processors, use set_handlers
 instead.
 
 =back
